src/streamlink/plugins/ltv_lsm_lv.py

```diff
@@ -1,45 +1,83 @@
 import logging
 import re
-from urllib.parse import urlparse
 
 from streamlink.plugin import Plugin, pluginmatcher
-from streamlink.plugin.api.utils import itertags
+from streamlink.plugin.api import validate
 from streamlink.stream.hls import HLSStream
 
 log = logging.getLogger(__name__)
 
 
 @pluginmatcher(re.compile(
-    r"https?://ltv\.lsm\.lv/lv/tieshraide"
+    r"https://ltv\.lsm\.lv/lv/tiesraide"
 ))
 class LtvLsmLv(Plugin):
     """
     Support for Latvian live channels streams on ltv.lsm.lv
     """
+
+    API_URL = "https://player.cloudycdn.services/player/ltvlive/channel/{channel_id}/"
+    _json_data_re = re.compile(r'teliaPlayer\((\{.*?\})\);', re.DOTALL)
+
+    main_page_schema = validate.Schema(
+        validate.parse_html(),
+        validate.xml_xpath_string(".//iframe[contains(@src, 'ltv.lsm.lv/embed')][1]/@src"),
+        validate.url()
+    )
+
+    embed_code_schema = validate.Schema(
+        validate.parse_html(),
+        validate.xml_xpath_string(".//live/@*"),
+        validate.parse_json(),
+        {"source": {"embed_code": validate.text}},
+        validate.get("source"), validate.get("embed_code"),
```

Comment:
`validate.get` shouldn't be chained, as it can receive nested keys via a tuple parameter nowadays:
`validate.get(("source", "embed_code"))`
Same down below...